src/world/World.php

```diff
@@ -1752,6 +1871,28 @@ public function getSkyLightReduction() : int{
 		return $this->skyLightReduction;
 	}
 
+	public function getRainLevel() : float{
+		return $this->rainLevel;
+	}
+
+	public function setRainLevel(float $level) : void {
+		if($level < 0.0 || $level > 1.0){
+			throw new \InvalidArgumentException("Rain level must be between 0.0 and 1.0, $level given");
+		}
+		$this->rainLevel = $level;
+	}
+
+	public function getThunderLevel() : float{
+		return $this->thunderLevel;
+	}
+
+	public function setThunderLevel(float $level) : void {
+		if($level < 0.0 || $level > 1.0){
+			throw new \InvalidArgumentException("Thunder level must be between 0.0 and 1.0, $level given");
+		}
+		$this->thunderLevel = $level;
+	}
```

Comment:
Why are these functions in a different place to the rest of the weather-related functions?